src/components/other/TabularBook/index.js

```diff
@@ -772,49 +775,53 @@ export default class TabularBook extends React.PureComponent {
 
         return (
             <div className={className}>
-                <ModalHeader
-                    title={_ts('tabular', 'title')}
-                    rightComponent={
-                        <div className={styles.headerContainer}>
-                            { isSomePending &&
-                                <div className={styles.pendingMessage}>
-                                    {
-                                        // Saving...
-                                        _ts('tabular', 'tabularSavingMessage')
-                                    }
-                                </div>
-                            }
-                        </div>
-                    }
-                />
+                {isModal &&
```

Comment:
IMO, instead of
```{isModal && <header>}
<body>
{isModal && <footer>}```
this could be a better approach
```
{isModal && <header><body><footer>}
{!isModal && <body>}
```